modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -681,4 +681,163 @@
 			font-size: 14px;
 		}
 	}
+}
+
+.col-md-4 {
```

Comment:
This fixes Calendar portlet styling when in a 30 column. After discussing this with @brandizzi, we thought styling it like the full page mobile calendar would be the best solution.